src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
```

Comment:
Just for convention, could we not use the @author tag? I just feel like it's a bit divisive, especially once we've all been working on all of the codebase. Maybe I'm weird, but that's always been how I've felt. What do you think?